visual-tests/axis.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>D3FC Visual Tests</title>
+    <link href="assets/style.css" rel="stylesheet">
+    <link href="../dist/d3-financial-components.css" rel="stylesheet">
+  </head>
+  <body>
+    <h1>D3FC Visual Tests</h1>
+
+    <section>
+        scale: <input type="range" min="1" max="1000" value="1" id="scale"></input>
+        offset: <input type="range" min="1" max="50" value="1" id="offset"></input>
+        ticks: <input type="range" min="5" max="50" value="5" id="ticks"></input>
+        <input type="checkbox" id="nice">nice</input>
+
+        <div>
+            <svg id="chart" width="900" height="450">
+        </div>
+    </section>
+
+    <script src="//d3js.org/d3.v3.js"></script>
+    <script src="../dist/d3-financial-components.js"></script>
+    
+
+    <script>
```

Comment:
Currently, JSHint only checks .js files (not .html).

Might it be worth moving the script into its own .js file? Or adding .html files to JSHint?

Very minor point, but there is a mix of double quotes and single quotes; there are a couple of missing semicolons too (lines 83 and 130).